scripts/valhalla_build_config

```diff
@@ -91,7 +91,8 @@ config = {
     'source_to_target_algorithm': 'select_optimal',
     'service': {
       'proxy': 'ipc:///tmp/thor'
-    }
+    },
+    'max_reserved_labels_count': 1000000
```

Comment:
according to https://github.com/valhalla/valhalla/issues/272#issue-204885361 it's about ~70 Mb